recipe/yarn.php

```diff
@@ -19,5 +19,5 @@
             run('cp -R {{previous_release}}/node_modules {{release_path}}');
         }
     }
-    run("cd {{release_path}} && {{bin/yarn}}");
+    run("cd {{release_path}} && {{bin/yarn}} --network-timeout 500000");
```

Comment:
imo it should be configurable :) like in rsync recipe :)

Reply:
Precisely! 😊